src/machine/machinectl.c

```diff
@@ -2666,18 +2666,22 @@ static int clean_images(int argc, char *argv[], void *userdata) {
                 return bus_log_parse_error(r);
 
         while ((r = sd_bus_message_read(reply, "(st)", &name, &usage)) > 0) {
-                log_info("Removed image '%s'. Freed exclusive disk space: %s",
-                         name, format_bytes(fb, sizeof(fb), usage));
+                if (usage == UINT64_MAX)
+                        log_info("Removed image '%s'", name);
+                else {
+                        log_info("Removed image '%s'. Freed exclusive disk space: %s",
+                                 name, format_bytes(fb, sizeof(fb), usage));
 
-                total += usage;
+                        total += usage;
+                }
                 c++;
         }
 
         r = sd_bus_message_exit_container(reply);
         if (r < 0)
                 return bus_log_parse_error(r);
 
-        log_info("Removed %u images in total. Total freed exclusive disk space %s.",
+        log_info("Removed %u images in total. Total freed exclusive disk space: %s.",
```

Comment:
Hmm, I think we should tweak this a bit: if we don't know how much disk space was removed in total, we shouldn't mention this either. Hence, I think `total` should be initialized to UINT64_MAX initially, and then in the loop we should have this:

```c
if (total == UINT64_MAX)
        total = usage;
else
        total += usage;
```

that way, if total is UINT64_MAX at the end of the loop we don't know how much we cleaned up, and it if itsn't we know that at least this much was cleared.